kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -191,6 +191,12 @@ public void checkpointed(long l)
   }
 
   @Override
+  public void beforeCheckpoint(long windowId)
```

Comment:
@chaithu14 what is the purpose of this change? Why add an interface with no implementation?